plugin/Seeder/Command.php

```diff
@@ -2,8 +2,8 @@
 
 namespace WP_Cypress\Seeder;
 
-use Exception;
 use WP_CLI;
+use Exception;
```

Comment:
@liamdefty just out of curiosity, did your IDE order these? 🤔

Reply:
Oops, not sure what happened there. Have re-ordered alphabetically.